src/tcgwars/logic/impl/gen4/Platinum.groovy

```diff
@@ -3509,8 +3402,8 @@ public enum Platinum implements LogicCardInfo {
             def pcs = my.all.findAll { it.topPokemonCard.cardTypes.is(POKEMON_SP) }.select("Which Pokémon SP to return to hand?")
             scoopUpPokemon(pcs, delegate)
           }
-          playRequirement{
-            my.bench.any { it.topPokemonCard.cardTypes.is(POKEMON_SP) }
+          playRequirement {
+            assert my.all.any { it.topPokemonCard.cardTypes.is(POKEMON_SP) } : "No Pokémon SP cards in play"
```

Comment:
This could be replaced with `assertMyAll(hasVariants: POKEMON_SP)`

But would require editing `assertAnyPokemonInPlay` in TcgStatics to support Pokémon SP

  * Adding `(CardType.POKEMON_SP):  { it.pokemonSP},` to variantFilters
  * Adding `(CardType.POKEMON_SP):  (CardType.POKEMON_SP).toString(),` to variantLabels